e2e/cypress/tests/integration/enterprise/system_console/channel_moderation/channel_mentions_spec.ts

```diff
@@ -24,10 +26,11 @@ import {
 } from './helpers';
 
 describe('MM-23102 - Channel Moderation - Channel Mentions', () => {
-    let regularUser;
-    let guestUser;
-    let testTeam;
-    let testChannel;
+    let regularUser: UserProfile;
+    let guestUser: UserProfile;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let testTeam: any;
```

Comment:
Hi @adithyaakrishna ,
 can we use Cypress.team and remove line 31

cypress.Chanel on line 33

Reply:
Done 💯